install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -302,6 +302,7 @@ openshift_cloudprovider_openstack_password: "{{ lookup('env','OS_PASSWORD') }}"
 openshift_cloudprovider_openstack_tenant_name: "{{ lookup('env','OS_PROJECT_NAME') }}"
 openshift_cloudprovider_openstack_blockstorage_version: v2
 openshift_cloudprovider_openstack_domain_name: "{{ lookup('env','OS_USER_DOMAIN_NAME') }}"
+openshift_cloudprovider_openstack_conf_file: PATH_TO_LOCAL_OPENSTACK_CONFIGURATION_FILE
```

Comment:
Is `PATH_TO_LOCAL_OPENSTACK_CONFIGURATION_FILE` an actual env variable, or is it something the user needs to replace? If the user needs to replace, then shouldn't it be lowecase and in angle brackets?

Reply:
Got the proposed new style in my head. Subbing.